git_machete/git_operations.py

```diff
@@ -111,15 +111,15 @@ def get_root_dir(self) -> str:
             try:
                 self.root_dir = self.popen_git("rev-parse", "--show-toplevel").strip()
             except MacheteException:
-                raise MacheteException("Not a git repository")
+                raise MacheteException("Not a git repository", verbose=False)
```

Comment:
Hmm okay... why specifically these 3 occurrences of `MacheteException` are created with `verbose=False`? Shouldn't `verbose=False` be the default actually?

Reply:
ok, that makes more sense :)